typescript/web/src/components/workspaces/update-plan-banner.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, chakra, Flex, Text } from "@chakra-ui/react";
+import { useRouter } from "next/router";
+import { RiInformationFill } from "react-icons/ri";
+import { useWorkspace } from "../../hooks/use-user";
+
+const InfoIcon = chakra(RiInformationFill);
+export const UpdatePlanBanner = () => {
```

Comment:
![image](https://user-images.githubusercontent.com/2091902/155528892-3bca66d5-b722-4ff7-8bc6-4e6fe4c4c414.png)

IDK if we talked about it yet in DSM or other but you can install the CodeClimate Chrome extension to see such lines in GitHub diff 😉

Reply:
Based on the discussion we had with @nicOwlas today, shouldn't we leave code metrics to the side for the moment as it's not that critical right now ?

Reply:
Have I already blocked a PR just because of CodeClimate feedbacks?
No because it's already on the side, It does not blocks the CI and we don't consider the warnings which involves too much time to be fixed.

One of the point of using CodeClimate is to help us getting code-quality feedbacks quicker than human code-reviews. It does not replaces them but avoids the need to manually tell you the obvious code smells.